src/config/adapters/dhis_highcharts/index.js

```diff
@@ -77,7 +77,7 @@ export default function ({ store, layout, el, extraConfig, extraOptions }) {
 
     // DHIS2-1243 add trend lines after sorting
     // trend line on pie and gauge does not make sense
-    if (layout.type !== CHART_TYPE_GAUGE && layout.type !== CHART_TYPE_PIE && layout.regressionType !== 'NONE') {
+    if (![CHART_TYPE_GAUGE, CHART_TYPE_PIE].includes(String(layout.type).toUpperCase()) && layout.regressionType !== 'NONE') {
```

Comment:
I remember issues with `includes` in IE, not sure if it was on Array or String tho...